clouddriver-ecs/src/main/java/com/netflix/spinnakerclouddriver/aws/ecs/EcsCloudProvider.java

```diff
@@ -0,0 +1,34 @@
+package com.netflix.spinnakerclouddriver.aws.ecs;
+
+import org.springframework.stereotype.Component;
+import com.netflix.spinnaker.clouddriver.core.CloudProvider;
+
+import java.lang.annotation.Annotation;
+
+//@Component
```

Comment:
I would prefer to keep the module itself totally intact, but remove it from the `clouddriver-web` dependencies as Tomas suggested. This will reduce cognitive load of what needs to happen when we're all finally comfortable with the state of the module, it also means you can do local development more easily.